packages/e2e-tests/helpers/console-panel.ts

```diff
@@ -220,12 +220,32 @@ export async function verifyConsoleMessage(
   );
 
   const messages = await findConsoleMessage(page, expected, messageType);
+  await verifyExpectedCount(messages, expectedCount);
+}
+
+export async function verifyEvaluationResult(
+  page: Page,
+  expected: Expected,
+  expectedCount?: number
+) {
+  await debugPrint(
+    page,
+    `Verifying the presence of an evaluation result with text "${chalk.bold(expected)}"`,
+    "verifyConsoleMessage"
+  );
+
+  const messages = page.locator(
+    `[data-test-name="TerminalExpression-Result"]:has-text('${expected}')`
+  );
+  await verifyExpectedCount(messages, expectedCount);
+}
 
+export async function verifyExpectedCount(locator: Locator, expectedCount?: number) {
```

Comment:
nit: Seems odd to me that we have a method named `verifyExpectedCount` that takes an _optional_ `expectedCount` param. What does it mean to call this method without that param?

Seems like we should just adjust the call sight in `verifyEvaluationResult` to only call this if the count is passed in.